src/components/ebay-infotip/test/test.browser.js

```diff
@@ -19,30 +19,43 @@ describe('given the default infotip', () => {
 
     afterEach(() => widget.destroy());
 
-    describe('when the host element is clicked', () => {
-        let spy;
-        beforeEach(() => {
-            spy = sinon.spy();
-            widget.on('tooltip-expand', spy);
-            testUtils.triggerEvent(host, 'click');
-        });
+    thenItCanBeOpenAndClosed();
```

Comment:
Do we not need one for the initial render?

Reply:
This line is what is testing the initial render.

https://github.com/eBay/ebayui-core/pull/602/files/3bc8f3b64af7c0b7fa61126544811f7eb1450cc7#diff-eab84de4b65d4fa0081b4cd28ee4eccfR30 is testing the rerender.

Reply:
Must have missed that second one. Thanks!